src/model.py

```diff
@@ -238,6 +263,22 @@ async def generate(self, request):
                 stream = stream.as_numpy()[0]
             else:
                 stream = False
+            exclude_input_in_output = pb_utils.get_input_tensor_by_name(
```

Comment:
```suggestion
            prepend_input = not pb_utils.get_input_tensor_by_name(
```

I think in this case - I would recommend renaming the variable internally with a note.